app/src/main/java/com/example/life_is_well/HealthPageMain.kt

```diff
@@ -1,11 +1,29 @@
 package com.example.life_is_well
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import androidx.core.content.ContextCompat.startActivity
 
 class HealthPageMain : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_health_page_main)
+        setContentView(R.layout.activity_main)
+
+        val macrosButton = findViewById<Button>(R.id.macroTrackerButton)
+        macrosButton.setOnClickListener {
+            val Intent = Intent(this, PhysicalMacroTracker::class.java)
+            startActivity(Intent)
+
+        }
     }
-}
\ No newline at end of file
+}
+    /*fun healthBackBtn(view: View) {
+
+        val intent = Intent(this, MainActivity::class.java )
+        startActivity(intent)
+        }
+    */
```

Comment:
This would be a good example of a GitHub issue you could put in. You can also add a `TODO` comment in here about what you want to implement and link the issue number to it.